src/SemConv/composer.json

```diff
@@ -26,7 +26,8 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-main": "1.x-dev"
+            "dev-main": "1.x-dev",
+            "dev-2.x": "2.x-dev"
```

Comment:
The version of this package is based on the https://github.com/open-telemetry/semantic-conventions version? -> should stay on `1.x`.